frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -269,11 +293,7 @@ export const VMSchedulingList: React.FC<VMSchedulingListProps> = ({
   const isVM = kindObj === VirtualMachineModel;
   const vmiLike = isVM ? vm : vmi;
   const vmiLikeWrapper = asVMILikeWrapper(vmiLike);
-  const canEdit =
-    vmiLike &&
-    canUpdateVM &&
-    kindObj !== VirtualMachineInstanceModel &&
-    !isVMRunningOrExpectedRunning(vm);
+  const canEdit = vmiLike && canUpdateVM && kindObj !== VirtualMachineInstanceModel;
```

Comment:
should we allow edit for the other fields as well even though they are not part of this PR?

Reply:
I think we should, there should be no discrimination between fields in this regard

Reply:
everything on vm details Id say

Reply:
the issue is that we are not advertising that a restart is needed for these other fields in this PR